vendor.mod

```diff
@@ -129,17 +130,24 @@ require (
 	github.com/hashicorp/go-msgpack v0.5.5 // indirect
 	github.com/hashicorp/go-sockaddr v1.0.2 // indirect
 	github.com/hashicorp/golang-lru v0.5.4 // indirect
+	github.com/in-toto/in-toto-golang v0.3.4-0.20220709202702-fa494aaa0add // indirect
```

Comment:
Opened https://github.com/moby/buildkit/pull/3415, but it looks like BuildKit depends on things that are not compatible with current releases.